### ISSUE TITLE
Malformed OpenTopic url when creating a publisher

### ISSUE THREAD

**Issue by thmshmm** [devopsfaith/krakend-pubsub#4]
When trying to initialize a pubsub publisher, the OpenTopic gets called with the host DNS and topic_url parameters concatenated. As described in [gocloud/pubsub](https://gocloud.dev/howto/pubsub/publish/#services) the schema should look like schema://topicXY.

`ERROR pubsub: open topic kafka://kafka-1:9092kafka://test-in: KAFKA_BROKERS environment variable not set`

The debug error shows the malformed topic parameter. Additionally I think the host parameter of the backend could automatically be converted into the necessary environment variable for the read by the publisher.

**Comment by kpacha** [comment]
did you try something like this?

```
"host": ["kafka://"],
"extra_config":{
"github.com/devopsfaith/krakend-pubsub/publisher": {
    "topic_url": "test-in"
}
}
```

and passing the host `kafka-1:9092` to the `KAFKA_BROKERS` envar

**Comment by thmshmm** [comment]
Tested it with the newly released 1.1.0 and with your settings it is working as expected. Thanks.
